tests/integration_tests/performance/test_vsock_throughput.py

```diff
@@ -203,8 +203,8 @@ def consume_iperf_output(cons, result):
         cpu_util_host = cpu_util[CPU_UTILIZATION_VMM]
         cpu_util_guest = cpu_util[CPU_UTILIZATION_VCPUS_TOTAL]
 
-        cons.consume_stat("value", CPU_UTILIZATION_VMM, cpu_util_host)
-        cons.consume_stat("value", CPU_UTILIZATION_VCPUS_TOTAL, cpu_util_guest)
+        cons.consume_stat("Avg", CPU_UTILIZATION_VMM, cpu_util_host)
+        cons.consume_stat("Avg", CPU_UTILIZATION_VCPUS_TOTAL, cpu_util_guest)
```

Comment:
why not name the stat `total`?

Reply:
Since this is a ValuePlaceholder, I would keep the name the way it was: `value`. the associated measurement name is explicit enough. However, we can consider this a nit